### PR TITLE
[#93705874] OpenConnect ansible role.

### PR DESCRIPTION
**Pull request by mtekel**
[Create an authentication mechanism for the build service](https://www.pivotaltracker.com/story/show/93705874)

**What**

Create a simple ansible role to install and configure OpenConnect SSL VPN client.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+galaxy_info:
```
Comment (ajlanghorn) [inline meta/main.yml:2]: This whole section probably needs completing appropriately if it's going to get submitted to Galaxy, or perhaps removed if it's not.

**Comment by actionjack** [comment]
@ajlanghorn We've updated the meta file.

**Comment by ajlanghorn** [comment]
Great, thanks @actionjack. I would also consider changing the `author` value to `Government Digital Service`, or similar.

**Comment by government-paas-user** [comment]
Ah yes :+1: on author to GDS, thanks @ajlanghorn.

**Comment by government-paas-user** [comment]
NB I have not reviewed this, just weighing in to support Andrew's comment.

**Comment by actionjack** [comment]
@ajlanghorn @multicloud-user Author updated

**Comment by government-paas-user** [comment]
Sorry - that was Anna, logged in as multicloud-user :)

**Comment by ajlanghorn** [comment]
The only other thing I see is that 'company' value being set to 'Government Digital Service' or 'HM Government' or something so that you set the precedent now. But that's me being possibly a bit picky?

**Comment by actionjack** [comment]
@ajlanghorn Fixed!

**Comment by ajlanghorn** [comment]
@actionjack Cool. I'll leave a responsible adult to merge. Not sure I qualify ;)

**Comment by jimconner** [comment]
LGTM - Merging..

**Pull request merged**